core/src/io/Filter_File_Handle.cpp

```diff
@@ -139,9 +143,10 @@ namespace IO
         return true;
     }
 
-    void Filter_File_Handle::Read_String( std::string& var, const std::string keyword, 
-                                            bool log_notfound )
+    void Filter_File_Handle::Read_String( std::string& var, std::string keyword, bool log_notfound )
     {
+        std::transform( keyword.begin(), keyword.end(), keyword.begin(), ::tolower );
+        
         if ( Find( keyword ) )
         {
             getline( this->iss, var );

```

Comment:
This will not lowercase the read-in string, `var` should also be transformed after this.

Reply:
- The attribute of the `Filter_File_Handle` object `std::string line` is populated by the function `Filter_File_Handle::GetLine_Handle()` for each line of the file. In case that the whole line is not a comment, `line` is processed by a call in `Filter_File_Handle::GetLine()`. 
- The function `Filter_File_Handle::GetLine()` is __de-capitalizing__ every `line`.
- The `std::istringstream iss` is cleared and populated for each `line` of the input file by the call to  `Filter_File_Handle::Find_in_Line()`.
- We are getting `var` from attribute `iss`.
Hence the `var` will always hold a decapitalized string.